src/Components/UpcomingRow/UpcomingRow.tsx

```diff
@@ -75,63 +88,78 @@ export const TableRow: React.FunctionComponent<TableRowProps> = ({
   }
 
   return (
-    <Tbody isExpanded={isExpanded}>
-      <Tr>
-        <Td
-          expand={{
-            rowIndex: rowIndex,
-            isExpanded,
-            onToggle,
-            expandId: 'composable-expandable-example',
-          }}
-        />
-        <Td dataLabel={columnNames.name} modifier="truncate">
-          {repo.name}
-        </Td>
-        <Td dataLabel={columnNames.type} modifier="truncate">
-          <span className="drf-lifecycle__upcoming-row-type">
-            {typeIcon}
-            {repo.type}
-          </span>
-        </Td>
-        <Td dataLabel={columnNames.release} modifier="truncate">
-          {repo.release}
-        </Td>
-        <Td dataLabel={columnNames.date} modifier="truncate">
-          {repo.date}
-        </Td>
-      </Tr>
-      {repo.details ? (
-        <Tr isExpanded={isExpanded}>
-          {!childIsFullWidth ? <Td /> : null}
+    <>
```

Comment:
A lot of changes there is cause by adding a top element -> adding tab to all lines